src/promptflow/promptflow/_core/tracer.py

```diff
@@ -14,9 +14,10 @@
 from threading import Lock
 from typing import Callable, Dict, List, Optional
 
+from openai.types.chat.chat_completion import ChatCompletion
```

Comment:
If it is not openai v1, I think it will fail to import.

Reply:
Fixed.